File: problemas/1100/1174.php
<?php

$a = array_map(fn() => readline(), range(0, 100));

for ($i = 0; $i < 100; $i++) {
    if ($a[$i] <= 10) {
        echo "A[{$i}] = " . number_format($a[$i], 1, '.', '') . "\n";
    }
}

File: problemas/1100/1174.cs
using System;
using System.Linq;

namespace uri1174
{
    internal static class Program
    {
        private static void Main()
        {
            double[] a = Enumerable.Range(0, 100)
                .Select(_ => double.Parse(Console.ReadLine()))
                .ToArray();

            for (int i = 0; i < 100; i++)
            {
                if (a[i] <= 10)
                {
                    Console.WriteLine($"A[{i}] = {a[i]:F1}");
                }
            }
        }
    }
}